surveys/2024-04/questions.md

```diff
@@ -0,0 +1,160 @@
+  - With a Node.js version manager: nvm, n, nave, nvs, volta, etc.
+  - Using the official installer
+  - Using package managers like npm or yarn and install Node.js as a package
+  - Using a system package manager: apt-get, brew, dnf, yum …
+  - Downloading binaries directly from Nodejs.org
+  - Building Node.js from Source
+  - Other
+
+- What package manager do you use ? (multiple choice - Other should give an input)
+  - npm
+  - yarn
+  - cnpm
+  - pnpm
+  - Other?
+
+- If you use a version manager which one do you use (multiple choice - Other should give an input)
+  - nvm
+  - n
+  - nave
+  - nvs
+  - volta
+  - Other?
+
+- How do you manage the package manager for your project? (multiple choice – Other should give an input)
+  - I use one version installed globally for all my projects.
+  - I use a tool to pick a specific version per project (e.g. Corepack, asdf, …).
+  - I use containers.
+  - Other?
+
+## Project Priorities and Direction
+
+- Which of the current [Technical Priorities](https://github.com/nodejs/node/blob/main/doc/contributing/technical-priorities.md) are important to you ? (multiple choice)
+  - Modern HTTP
+  - Suitable types for end-users
+  - Documentation
+  - WebAssembly
+  - ESM
+  - Support for features from the latest ECMAScript spec
+  - Observability
+  - Permissions/policies/security model
+  - Better multithreaded support
+  - Single Executable Applications
+
+- Are there technical priorities that you believe are missing (open question)
+
+- What is important to you ? (multiple choice)
+  - Good understanding of the direction of the project
+  - Ability to affect the direction of the project
+  - Consumable APIs and docs
+  - Predictable and stable releases
+  - Innovation at a consumable pace
+  - Easy Installation
+  - Easy issue reporting, resolution and collaboration
+  - Broad deployment platform support
+  - Broad desktop platform support
+  - Consistent and intuitive error handling
+  - Runtime diagnostic tooling
+  - Development time diagnostic tooling
+  - Relevant APIs in core
+  - Module/dependency info and management
+  - Ways to fund their work
+  - Ability to assess impact of changes they make
+  - Reasonable resource usage/performance
+  - Good security and CVE practices in the project
+  - Better CVE management in the ecosystem
+  - Good CI infrastructure and experience in the project
+  - Supportive Collaborators and Environment in the project
+  - Better ways to build consensus in the project
+  - Easy contribution workflow
+  - Ability to embed and bundle the Node.js runtime
+  - A well maintained and secure standard library
+  - Assets that show Node.js is a good choice
+
+- What is important to you that is not in this list? (open question)
+
+## Technical Questions
+
+- Regardless of how your code is written in its original form, when it runs in production, does it contain ES module syntax (`import`/`export`)?
```

Comment:
I think this may not be clear enough for those who using a transpiler to run faux ESM and they might still read it as "do you write your code in `import`/`export`"....(as many do not even understand what faux ESM is despite having been running it).

A more practical question may be:

```suggestion
- In your application, is `process.mainModule` undefined?
```

Reply:
@joyeecheung I believe your suggestion is too technical, and not many people will be able to answer